source/connect/looker.txt

```diff
@@ -0,0 +1,35 @@
+:noprevnext:
+
+===================
+Connect from Looker
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can import data from a MongoDB collection into the Looker BI Platform
+with the {+download-center-bi+}. You do not need an ODBC data connection or
+a System DSN.
+
+Prerequisites
+-------------
+
+Connecting the Looker BI Platform to the |bi-atlas| requires the
+following:
+
+- An M10+ Atlas cluster with the |bi-short| enabled
+
+- A `Looker <https://looker.com/product/business-intelligence>`_ account
```

Comment:
Thought: I'm not a fan of bulleted list starting with a/an/etc.  Perhaps it wouldn't read as cleanly though. It might look better, thoughts?

Reply:
Fixed.